components/common/listingCart/ListingCartMini.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <transition v-if="listingCartStore.count" name="slide">
+    <div class="listing-container">
+      <div class="is-inline-flex is-align-items-center">
+        <div class="k-shadow theme-background-color border py-2">
+          <div class="is-inline-flex is-align-items-center mx-5">
+            <div>
+              <b>{{ listingCartStore.count }}</b> {{ $t('items') }}
+            </div>
+            <div class="bar mx-4" />
```

Comment:
```suggestion
<div class="mx-4" />
```

bar class doesn't exist

Reply:
@daiagi There's a problem with h-full
<img width="494" alt="image" src="https://github.com/kodadot/nft-gallery/assets/3810177/39498e6e-be22-415f-b8a9-07cc6abe9695">
<img width="494" alt="image" src="https://github.com/kodadot/nft-gallery/assets/3810177/773e6d94-79cc-400b-80b3-119e301b4e73">



I have removed the unused style now though